index.js

```diff
@@ -5,6 +5,10 @@ const fp = require('fastify-plugin')
 const Next = require('next')
 
 function fastifyNext (fastify, options, next) {
+  fastify.register(require('under-pressure'), options.underPressure)
```

Comment:
We need to have an if to avoid installing this twice if it's already registered. We should also support `underPressure: false` to avoid adding it.

Reply:
ah yeah that makes sense. so under-pressure cannot be installed more than once? I thought it was possible to install it multiple times as long as the different registrations don't step on each other's feet.

having an option to disable it make sense to me. do you think it should be the default value? (of having it disabled I mean)